crates/go_to_line/src/cursor_position.rs

```diff
@@ -104,19 +133,29 @@ impl CursorPosition {
             lines,
             characters,
             selections,
+            words,
         } = self.selected_count;
         let format = LineIndicatorFormat::get(None, cx);
         let is_short_format = format == &LineIndicatorFormat::Short;
         let lines = (lines > 1).then_some((lines, "line"));
         let selections = (selections > 1).then_some((selections, "selection"));
         let characters = (characters > 0).then_some((characters, "character"));
+        let words = words
+            .map(|words| {
```

Comment:
This can just be a `words.filter(|words| words <= 0).map(|words| {(words, "word"})`